src/main/java/org/jabref/model/pdf/FileAnnotation.java

```diff
@@ -1,127 +1,116 @@
 package org.jabref.model.pdf;
 
 
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+
 import org.apache.pdfbox.cos.COSDictionary;
 import org.apache.pdfbox.cos.COSName;
 import org.apache.pdfbox.pdmodel.interactive.annotation.PDAnnotation;
 
-public class FileAnnotation {
+public final class FileAnnotation {
 
-    private String commentId;
-    private String author;
-    private String date;
-    private int page;
-    private String content;
-    private String annotationType;
-    private FileAnnotation linkedFileAnnotation;
     private final static int ABBREVIATED_ANNOTATION_NAME_LENGTH = 45;
+    private final static String ANNOTATION_DATE_REGEX = "D:\\d+";
+    public final String author;
```

Comment:
I know that this is not a problem of mutability here, but is there a good reason for breaking with the Java conventions for this and the following attributes? If there is no added benefit, then I would much prefer to have public getters instead of public attributes for the simple reason that this is the convention. Otherwise it always looks fishy when such a variable is accessed in another class and I have to look up `FileAnnotation` to check that this is not a grave access bug.